stl/inc/xmemory

```diff
@@ -1644,7 +1672,7 @@ _CONSTEXPR20 _NoThrowFwdIt _Uninitialized_move_unchecked(_InIt _First, const _In
     }
     _Uninitialized_backout<_NoThrowFwdIt> _Backout{_Dest};
     for (; _First != _Last; ++_First) {
-        _Backout._Emplace_back(_STD move(*_First));
+        _Backout._Emplace_back_deref_move(_First);

```

Comment:
Isn't this going to break `constexpr` usage when `*_First` is a prvalue since `_Construct_in_place_by_deref` isn't `constexpr`? (Ditto for `_Uninitialized_copy_unchecked`.) (Suggests lack of test coverage.) I wonder if we could hack around this by making `_Construct_in_place_by_deref` `_CONSTEXPR20` and using `_MSVC_CONSTEXPR` for the placement `new`.